pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = ""
 authors = ["Ought <team@ought.org>"]
 
 [tool.poetry.dependencies]
-python = "^3.6.8"
+python = "^3.8.5"
```

Comment:
Hmmm... I'm pretty sure `python = "^3.8.5"` means ">= 3.8.5 <4.0.0": https://python-poetry.org/docs/versions/

According to https://python-poetry.org/docs/versions/, the current code should mean ">= 3.6.8 <4.0.0", which should work

I wonder if the `.python-version` file, or some other file, is causing the error message ERROR: Package 'ergo' requires a different Python: 3.8.6 not in '>=3.6.8,<3.7.0' (really not sure though)